Docs/Tutorial, add comment on stores statefulness in SPAs (issue #4407) (#4610)
sveltejs/svelte: site/content/tutorial/08-stores/01-writable-stores/text.md

function reset() {
	count.set(0);
}
```

> In the particular case of Single Page Application (SPAs), different routes/pages are usually rendered as components. One could be tempted to use writable stores to keep global application state. Bear in mind, that in practice, reloading the page will reset the stores. 

Reviewer: "Single Page Application" should probably be "Single Page Applications"

alternatively, if you were saying just "Single Page Application" then you'd add an 'a' and remove an 's' to make it "a Single Page Application (SPA)"

> In the particuliar case of Single Page Application (SPAs), different routes/pages as usually rendered as components. One could be tempted to use writables stores to keep a global application state. Bear in mind that in practice, if users reload from their browser, store variables will be erased. 

Reviewer 2: ```suggestion
In the particuliar case of Single Page Application (SPAs), different routes/pages are usually rendered as components. One could be tempted to use writables stores to keep a global application state. Bear in mind that in practice, reloading the page will reset the stores. 
```

Reviewer 2: Seems good to me apart from minor fixes.

Reviewer 2: Updated the suggestion above, thanks Ben!
